From 2021e49c613106398be847d52e37c2148e715241 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Mon, 28 Nov 2016 13:33:24 +0200
Subject: [PATCH 1151/2241] fix: dts: a80x0: enable SDHCI driver for
 Armada-80x0-A

enable SDHCI for both AP/CP dies

Change-Id: I5088b0669b59e15ced7d1e387730477c600ad796
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/34131
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-db-A.dts      | 6 ++++++
 arch/arm64/boot/dts/marvell/armada-8040-db-router.dts | 3 +++
 2 files changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-A.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-A.dts
index dc33678..f35db4d 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-A.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-A.dts
@@ -61,6 +61,9 @@
 			serial@512000 {
 				status = "okay";
 			};
+			sdhci@6e0000 {
+				status = "okay";
+			};
 			i2c@511000 {
 				status = "disabled";
 			};
@@ -82,6 +85,9 @@
 			usb3h1: usb3@510000 {
 				status = "okay";
 			};
+			sdhci@780000 {
+				status = "okay";
+			};
 			mdio@12a200 {
 				#address-cells = <1>;
 				#size-cells = <0>;
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
index 3924e1c..c9ca277 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
@@ -61,6 +61,9 @@
 			serial@512000 {
 				status = "okay";
 			};
+			sdhci@6e0000 {
+				status = "disabled";
+			};
 			i2c@511000 {
 				status = "okay";
 				clock-frequency = <100000>;
-- 
2.7.4

